src/lib/public/board/types.ts

```diff
@@ -33,6 +33,7 @@ export enum Social {
   Discord = 'discord',
   LinkedIn = 'linkedin',
   Instagram = 'instagram',
+  Youtube = 'youtube',
```

Comment:
Nit: YouTube has a capital T.
```suggestion
  YouTube = 'youtube',
```